librz/core/cmd/cmd_debug.c

```diff
@@ -4410,23 +4410,67 @@ RZ_IPI RzCmdStatus rz_cmd_debug_add_bp_handler(RzCore *core, int argc, const cha
 
 // dbl
 RZ_IPI RzCmdStatus rz_cmd_debug_list_bp_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
-	RzOutputMode mode = state->mode;
-	switch (mode) {
-	case RZ_OUTPUT_MODE_STANDARD:
-		rz_bp_list(core->dbg->bp, 0);
-		break;
-	case RZ_OUTPUT_MODE_RIZIN:
-		rz_bp_list(core->dbg->bp, 1);
-		break;
-	case RZ_OUTPUT_MODE_JSON:
-		rz_bp_list(core->dbg->bp, 'j');
-		break;
-	case RZ_OUTPUT_MODE_QUIET:
-		rz_bp_list(core->dbg->bp, -1);
-		break;
-	default:
-		rz_warn_if_reached();
-		break;
+	RzBreakpointItem *b;
```

Comment:
Avoid this change if not strictly necessary. I think it has not anything to do with `c` porting.

Reply:
(saying because otherwise there are few things to fix here as well and they are not `c` related).

Reply:
So shall I move this commits to another branch, and then open a separate PR for this?

Reply:
yes please. there, i suggest to check if rz_bp_list is still needed, if not to delete it, then to use rz_cons_Printf instead of cb_printf.

Reply:
I have moved these commits in a new PR (#2012).